requirements.txt

```diff
@@ -1,220 +1,223 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --annotation-style=line --output-file=requirements.txt setup.py
+#    pip-compile --annotation-style=line --output-file=requirements.txt ./modules/reana-db/setup.py setup.py
```

Comment:
TODO: regenerate `requirements.txt` with the usual command after https://github.com/reanahub/reana-db/pull/234 has been merged and reana-db has been released